skeleton/pipeline.mk

```diff
@@ -41,6 +41,7 @@ generated-source:
 integration-test:
 	@echo "  >   Starting integration tests"
```

Comment:
Should we just replace the main `Makefile` `integration-test` with this wholesale?
```
@$(MAKE) start-containers
...
@$(MAKE) stop-containers
```

Reply:
Done